examples/source/interactivity-dynamic-content/SeatMap/api.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const express = require('express');
+const SampleRenderer = require('@examples/lib/SampleRenderer');
+const {createRequestContext} = require('@lib/templates/');
+const utils = require('@lib/utils');
+const seats = require(utils.project.absolute('/examples/static/samples/json/seats.json'));
+
+// eslint-disable-next-line new-cap
+const router = express.Router();
+
+SampleRenderer.use(router, (request, response, template) => {
+  response.send(template.render(createRequestContext(request, {
```

Comment:
Can't you simply pass seats `createRequestContext(request, seats)`?

Reply:
Oh yes, my bad. Just changed it.  Thanks for pointing it out.